packages/runtime/src/widgets/DataGrid/DataGrid.tsx

```diff
@@ -84,6 +84,7 @@ export type GridProps = {
   hidePagination?: boolean;
   scroll?: DataGridScrollable;
   onScrollEnd?: EnsembleAction;
+  onPageChange?: EnsembleAction;
```

Comment:
Let's add an attribute for pageSize and if they pass 0 or less it's equivalent to hiding pagination